cmd/oras/pull.go

```
@@ -201,11 +211,7 @@ func runPull(opts pullOptions) error {
 		}
 		for _, s := range successors {
 			if _, ok := s.Annotations[ocispec.AnnotationTitle]; ok {
-				if _, ok := printed.LoadOrStore(generateContentKey(s), true); !ok {
-					if err = display.PrintStatus(s, "Restored   ", opts.Verbose); err != nil {
-						return err
-					}
-				}
+				printOnce(s, "Restored   ", opts.Verbose, printed)
```

return error is not checked.